src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -334,6 +341,11 @@ private EditorForm getEditorForm(ExtendedNodeType primaryNodeType, Locale uiLoca
         }
     }
 
+    private boolean shouldHideSection(EditorFormSectionDefinition section, JCRNodeWrapper existingNode){
+        String currentMode = existingNode == null ? "create" : "edit";
```

Comment:
Can you add a variable in some way to avoid to do the same test a line 301 `existingNode == null`, then pass the mode directly to the function (instead of the node)

Reply:
Done!